assets/js/index.js

```diff
@@ -198,13 +206,24 @@ function updateDisplay() {
         if (i == code) {
           selected = true;
         }
-        text += `<option value="${i}" ${i == code ? 'selected': ''}>` + messageArray[j].name + '</option>';
+        var catText = categoryMap.get(messageArray[j].category);
+        var option = `<option value="${i}"${i == code ? ' selected': ''}>` + messageArray[j].name + '</option>';
+        // Append option to category
+        catText = !catText ? option : catText + option;
+        categoryMap.set(messageArray[j].category, catText);
```

Comment:
In the description of the PR you wrote:
> messages with undefined categories will not be shown in the dropdown menu

But isn't the line above setting even for undefined categories a text, and therefore the messages will appear in the dropdown? (haven't tested it yet)

Reply:
Ah, yep, they are assigned a text here. However, they're skipped on line 221, because only categories that are defined in the category list are added to the dropdown. Maybe I don't do this in the most efficient way.